components/toast/stories/template.js

```diff
@@ -63,3 +64,74 @@ export const Template = ({
 		</div>
 	`;
 };
+
+export const ToastWrapOptions = ({
+	...args
+}, context ) => Container({
+	withBorder: false,
+	direction: "column",
+	wrapperStyles: {
+		columnGap: "12",
```

Comment:
```suggestion
		columnGap: "12px",
```

This will need a unit added.